packages/scss/src/components/table/mods.scss

```diff
@@ -53,6 +53,40 @@
 @mixin card {
 	border-top: 0;
 
+	:where(.table-head, .table-body, .table-foot) {
+		&:first-child {
+			:where(.table-head-row, .table-body-row, .table-foot-row) {
+				&:first-child {
+					:where(.table-head-row-cell, .table-body-row-cell) {
+						&:first-child {
+							border-top-left-radius: var(--components-card-border-radius);
+						}
+
+						&:last-child {
+							border-top-right-radius: var(--components-card-border-radius);
+						}
+					}
+				}
+			}
+		}
+
+		&:last-child {
+			:where(.table-head-row, .table-body-row, .table-foot-row) {
+				&:last-child {
```

Comment:
if there is a table-foot it will be strange to have à border-radius on the table-body-row last-child

Reply:
There isn't always a `tfoot`, it can also be a `tbody`, but in both cases they have to be last-child.